examples/snowtooth/query-service.bal

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/graphql;
+import ballerina/lang.array;
+import snowtooth.datasource as ds;
+
+type SearchResult Lift|Trail;
+
+service /graphql on new graphql:Listener(9000) {
+
+    # A list of all `Lift` objects
+    # + return - the lifts
+    resource function get allLifts(ds:Status? status) returns Lift[] {
```

Comment:
There's a bug in compiler plugin validation. We have to fix it for this to work.
https://github.com/ballerina-platform/ballerina-standard-library/issues/1350